apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/CcwRifLoadJob.java

```diff
@@ -69,6 +71,12 @@
 public final class CcwRifLoadJob implements PipelineJob {
   private static final Logger LOGGER = LoggerFactory.getLogger(CcwRifLoadJob.class);
 
+  /**
+   * Maximum age of a manifest for it to be considered for processing. Intended to keep database
+   * queries reasonable.
+   */
+  public static final Duration MAX_MANIFEST_AGE = Duration.ofDays(90);
```

Comment:
being VERY generous here...I'd be surprised if we would ever exceed say 4-6 weeks!

Reply:
That's good to know.  I wanted to make it long enough it wouldn't bite someone who tries to load old data but short enough to have an impact on performance.  If you think90 is too long maybe 60 days would be better?  Or I could make it configurable.

Reply:
let's go with 60....quite honestly, if we are over 2 weeks, we'd be in a heap of trouble!